my-app/src/components/UserCard.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const UserCard = (props) => {
+
+    const genData = props.data.gitHubData;
+
+    return (
+        <div className="userCard">
```

Comment:
Looks great. You may want to adjust the styling of your avatar picture so that it doesnt' overtake the screen.